code/src/App.js

```diff
@@ -1,9 +1,89 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './app.css'
+import './index.css'
+
+
+const ageGroup = ['too young', '19-35', '35-65', '+65'];
+const option0 = ''
+const option1 = 'Candlelight dinner in a cosy restaurant'
+const option2 = 'Netlfix and Chill'
+const option3 = 'Dancing on my own'
+const option4 = 'Party with my friends'
+const option5 = 'Headbangers ball concert'
 
 export const App = () => {
+  const [ageGroups, setAgeGroup] = useState("");
+  const [hobby, setHobby] = useState("");
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [showSummary, setShowSummary] = useState(false);
+  const handleSubmit = event => {
+    event.preventDefault();
+    setShowSummary(true);
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className='form-container'>
+      <h1>Win a very special evening for you and someone equally as special!</h1>
+      <h3>Just fill in the form below to participate</h3>
+      <p>... and you could be the lucky one.</p>
+
+      <div className='questions-container'>
+        <div className="questions">
+
+          <form onSubmit={handleSubmit}>
+            <h2>What is your name?</h2>
+            <input type='text'
+              onChange={event => setName(event.target.value)}
+              value={name} placeholder="Your name">
+            </input>
+            <h2>{`How old are you ${name}?`}</h2>
+            {ageGroup.map(group => (
+              <label key={group}>
+                <input type='radio'
+                  value={group}
+                  onChange={event => setAgeGroup(event.target.value)}
+                  checked={ageGroups === group}
+                />
+                {group}
+              </label>
+            ))}
+
+            <h2> What is a perfect evenig according to you?</h2>
+            <select className="hobby"
+              onChange={event => setHobby(event.target.value)}
+              value={hobby}>
+              <option value={option0}>~ Select option ~</option>
```

Comment:
I guess you don't want this as an option? :)
```suggestion
              <option value={option0} disabled>~ Select option ~</option>
```